test/e2e/fixtures.go

```diff
@@ -151,14 +147,19 @@ func setupTestWithIPFIXCollector(tb testing.TB) (*TestData, error, bool) {
 	ipfixCollectorIP, err := testData.podWaitForIPs(defaultTimeout, "ipfix-collector", testNamespace)
 	if err != nil || len(ipfixCollectorIP.ipStrings) == 0 {
 		tb.Errorf("Error when waiting to get ipfix collector Pod IP: %v", err)
-		return nil, err, isIPv6
+		return nil, isIPv6, err
+	}
+	var ipStr string
+	if isIPv6 {
+		ipStr = ipfixCollectorIP.ipv6.String()
```

Comment:
I think it is better to check existence of `ipfixCollectorIP.ipv6` first since `isIPv6` is the result of `clusterInfo.podV6NetworkCIDR == ""`.
The same for IPv4.

Reply:
Since you didn't change L158, don't you need to wrap `<ipv6-addr>` with `[]`?
Or change L158 to:
`ipfixCollectorAddr := fmt.Sprintf("%s:tcp", net.JoinHostPort(ipStr, ipfixCollectorPort))`

Reply:
Thanks, addressed.